### PR TITLE
Add arp-scan package

### PR DESCRIPTION
**Pull request by urusha**

### CODE REVIEW

File: net/arp-scan/Makefile [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (C) 2006-2016 OpenWrt.org
```
Comment (yousong) [inline net/arp-scan/Makefile]: I am not a lawyer, but if this file is only introduced this year, then just `2016` should be used.

File: net/arp-scan/Makefile [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/royhills/arp-scan.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=69b2f70588565385f3a61d9b1f100f01747cd5e1
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+
+PKG_FIXUP:=autoreconf
+
+PKG_BUILD_DEPENDS:=libpcap
```
Comment (yousong) [inline net/arp-scan/Makefile]: If arp-scan links dynamically against libpcap, then this line can be removed.

**Comment by urusha** [comment]
Yes, it's just a hack, but arp-scan compiles and works just fine (at least on rt305x). Please, review fixed version.

**Comment by yousong** [comment]
thanks

**Pull request merged**